WolvenKit.Modkit/RED4/Tasks/UncookTask.cs

```diff
@@ -38,19 +38,20 @@ public int UncookTask(FileSystemInfo[] paths, UncookTaskOptions options)
             return ERROR_BAD_ARGUMENTS;
         }
 
-        if (options.outpath == null)
-        {
-            _loggerService.Error("Please fill in an output path.");
-            return ERROR_BAD_ARGUMENTS;
-        }
 
         if (options.meshExportType != null && string.IsNullOrEmpty(options.meshExportMaterialRepo) && options.outpath is null)
         {
             _loggerService.Error("When using --mesh-export-type, the --outpath or the --mesh-export-material-repo must be specified.");
             return ERROR_INVALID_COMMAND_LINE;
         }
 
-        if (options.gamepath != null && options.gamepath.Exists)
+        if (options.outpath == null)
```

Comment:
Same "problem" as before. The analyzer would still complain that there is a double null check (L42)